Exercises/objects/objects.js

```diff
@@ -0,0 +1,50 @@
+/*eslint-disable no-param-reassign*/
+//1. Create a object and using `defineProperty` define not writable property, not enumerable property and not
+//  configurable property. (PS: in real programming life I think you will never use it :) just for you to know)
+const car = {};
+Object.defineProperty(car, 'properties', {
+  doors: 4,
+  type: 'mechanical',
```

Comment:
Third parameter of `defineProperty` can have next keys:

configurable
enumerable
value
writable

All the rest will be ignored

In you case value of created key will be undefined

Reply:
(also get/set keywords are allowed to use here)